demos/object_detection_demo_ssd_async/main.cpp

```diff
@@ -50,523 +135,127 @@ bool ParseAndCheckCommandLine(int argc, char *argv[]) {
     return true;
 }
 
-void frameToBlob(const cv::Mat& frame,
-                 InferRequest::Ptr& inferRequest,
-                 const std::string& inputName) {
-    if (FLAGS_auto_resize) {
-        /* Just set input blob containing read image. Resize and layout conversion will be done automatically */
-        inferRequest->SetBlob(inputName, wrapMat2Blob(frame));
-    } else {
-        /* Resize and copy data from the image to the input blob */
-        Blob::Ptr frameBlob = inferRequest->GetBlob(inputName);
-        matU8ToBlob<uint8_t>(frame, frameBlob);
-    }
-}
+void paintInfo(cv::Mat& frame, const PipelineBase::PerformanceInfo& info) {
+    std::ostringstream out;
 
-enum class ExecutionMode {USER_SPECIFIED, MIN_LATENCY};
+    out.str("");
+    out << "FPS:" << std::fixed << std::setprecision(0) << std::setw(3) << info.movingAverageFPS
+        << " (" << std::setprecision(1) << info.FPS << ")";
+    putHighlightedText(frame, out.str(), cv::Point2f(10, 22), cv::FONT_HERSHEY_TRIPLEX, 0.6, cv::Scalar(10, 200, 10), 2);
 
-ExecutionMode getOtherMode(ExecutionMode mode) {
-    return mode == ExecutionMode::USER_SPECIFIED ? ExecutionMode::MIN_LATENCY : ExecutionMode::USER_SPECIFIED;
-}
+    out.str("");
+    out << "Avg Latency:" << std::fixed << std::setprecision(0) << std::setw(4) << info.movingAverageLatencyMs
+        << " (" << std::setprecision(1) << info.getTotalAverageLatencyMs() << ") ms";
+    putHighlightedText(frame, out.str(), cv::Point2f(10, 44), cv::FONT_HERSHEY_TRIPLEX, 0.6, cv::Scalar(200, 10, 10), 2);
 
-void switchMode(ExecutionMode& mode) {
-    mode = getOtherMode(mode);
+    out.str("");
+    out << "Inference Latency:" << std::fixed << std::setprecision(0) << std::setw(4) << info.getLastInferenceLatencyMs() << " ms";
+    putHighlightedText(frame, out.str(), cv::Point2f(10, 66), cv::FONT_HERSHEY_TRIPLEX, 0.6, cv::Scalar(200, 10, 10), 2);
+
+    out.str("");
+    out << "Pool: " << std::fixed << std::setprecision(1) <<
+        info.numRequestsInUse << "/" << FLAGS_nireq;
+    putHighlightedText(frame, out.str(), cv::Point2f(10, 88), cv::FONT_HERSHEY_TRIPLEX, 0.6, cv::Scalar(200, 10, 10), 2);
 }
 
 int main(int argc, char *argv[]) {
     try {
         /** This demo covers certain topology and cannot be generalized for any object detection **/
-        std::cout << "InferenceEngine: " << *GetInferenceEngineVersion() << std::endl;
+        std::cout << "InferenceEngine: " << *InferenceEngine::GetInferenceEngineVersion() << std::endl;
```

Comment:
There is a different way of printing the version now

Reply:
could you clarify ? All other examples use the same way.

Reply:
fixed